src/Take.Blip.Builder.UnitTests/AttendanceRedirectHandlerTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Lime.Protocol;
+using NSubstitute;
+using NSubstitute.ReceivedExtensions;
+using Take.Blip.Builder.Hosting;
+using Take.Blip.Client.Extensions.HelpDesk;
+using Takenet.Iris.Messaging.Resources;
+using Xunit;
+
+namespace Take.Blip.Builder.UnitTests
+{
+    public class AttendanceRedirectHandlerFixture : IDisposable
+    {
+        public IContext Context { get; private set; }
+        public IAttendanceRedirectHandler AttendanceRedirectHandler { get; private set; }
+        public IHelpDeskExtension HelpDeskExtension { get; private set; }
+        public IConfiguration Configuration { get; private set; }
+        public Identity FromIdentity { get; private set; }
+        public Identity UserIdentity { get; private set; }
+        public CancellationToken CancellationToken { get; private set; }
+
+        public AttendanceRedirectHandlerFixture()
+        {
+            Context = Substitute.For<IContext>();
+            HelpDeskExtension = Substitute.For<IHelpDeskExtension>();
+            Configuration = Substitute.For<IConfiguration>();
+
+            AttendanceRedirectHandler = new AttendanceRedirectHandler(HelpDeskExtension, Configuration);
+
+            FromIdentity = new Identity() { Domain = "msging.net", Name = "test" };
+            UserIdentity = new Identity() { Domain = "msging.net", Name = "user" };
+
+            CancellationToken = default;
+        }
+
+        public void Dispose()
```

Comment:
Is this being called? Do you have to implement `IDisposable`?

Reply:
I'd to implement it for using IClassFixture. As I used Async tests, its a good practice for XUnit to use it.